server/services/emails/email.js

```diff
@@ -0,0 +1,57 @@
+const Queue = require("bull");
+const Arena = require("bull-arena");
+const sgMail = require("@sendgrid/mail");
+require("dotenv").config();
```

Comment:
If we're using this file in server.js with an app.use(), we won't need this line